susemanager-utils/susemanager-sls/src/modules/mgrclusters.py

```diff
@@ -49,6 +49,9 @@ def upgrade_cluster(provider_module, params):
     fun = _get_provider_fun(provider_module, 'upgrade_cluster')
     return fun(**params)
 
+def upgrade_cluster_plan(provider_module, params):
+    fun = _get_provider_fun(provider_module, 'upgrade_cluster_plan')
+    return fun(**params)
```

Comment:
This shouldn't be required in case we simply use `upgrade_cluster(plan=True, **)`